spring/spring4/spring4-remote/src/test/java/org/infinispan/spring/provider/sample/SampleHotrodServerLifecycleBean.java

```diff
@@ -46,8 +47,8 @@ public void afterPropertiesSet() throws Exception {
       cacheManager.defineConfiguration(remoteBackupCacheName, HotRodTestingUtil.hotRodCacheConfiguration().build());
       cacheManager.defineConfiguration(customCacheName, HotRodTestingUtil.hotRodCacheConfiguration().build());
       HotRodServerConfigurationBuilder hcb = new HotRodServerConfigurationBuilder();
-      hcb.port(15233);
-      hotrodServer = HotRodClientTestingUtil.startHotRodServer(cacheManager, hcb);
+      hcb.port(PORT);
+      hotrodServer = HotRodClientTestingUtil.startHotRodServer(cacheManager, PORT, hcb);
```

Comment:
No, the 2 argument `#startHotRodServer` [does that](https://github.com/infinispan/infinispan/blob/044b850305703673f5ffca77a3ccfd4d2b9939d6/client/hotrod-client/src/test/java/org/infinispan/client/hotrod/test/HotRodClientTestingUtil.java#L50). That's why this test was broken :(